docs/installation.rst

```diff
@@ -92,7 +92,7 @@ Within the :file:`docs/` directory is a small Python script called
 
 .. prompt:: bash
 
-    python sphinx_server.py
+    poetry run python sphinx_server.py
```

Comment:
I haven't checked this locally myself, but I would have assumed that poetry is not required to be able to run the Sphinx server? e.g. if the user had a pip install

Reply:
Use of poetry here is not *required* per se, but it more of a "this should just work" suggestion given that we are assuming poetry for build/environment management for developers. A developer *can* of course create their own virtualenv/conda environment, but comes with assumed environment setup that is out of scope to detail here.

Reply:
If it wasn't clear above, I support Roddy's fix here.